src/sidebar/test/get-api-url-test.js

```diff
@@ -1,6 +1,6 @@
 import getApiUrl from '../get-api-url';
 
-describe('sidebar.getApiUrl', function () {
+describe('sidebar/getApiUrl', function () {
```

Comment:
```suggestion
describe('sidebar/get-api-url', function () {
```

`describe` blocks either reference just the name of the function/method if "reasonably unique" or the file path relative to `src/` and minus the `.js` extension.